### DISCUSSION TITLE
How can I edit default template

### DISCUSSION THREAD

**Discussion by Aremstrom** [open-webui/open-webui#1243] [Q&A] [answered]
I'm trying to customize the default templates login,singup,main page, specifically to change the name, title, etc. Could someone please provide guidance on how to do this? I've looked through the documentation, but I couldn't find clear instructions on how to proceed. Any help would be greatly appreciated. Thanks!

**Comment by changchiyou** [comment, marked as answer]
- https://github.com/open-webui/open-webui/issues/1238

---

- https://github.com/open-webui/open-webui/pull/880

https://github.com/open-webui/open-webui/blob/2fa94956f4e500bf5c42263124c758d8613ee05e/backend/config.py#L107-L129
